packages/desktop/views/onboarding/views/CongratulationsView.svelte

```diff
@@ -24,7 +23,7 @@
             checkOrConnectLedger(_continue)
         } else {
             updateOnboardingProfile({
-                strongholdVersion: features.onboarding.strongholdVersionCheck.enabled ? STRONGHOLD_VERSION : undefined,
+                strongholdVersion: STRONGHOLD_VERSION,
```

Comment:
This is fine here, but would make more sense to be on the build initial onboarding profile object